practices/managing-node-projects.md

```diff
@@ -34,23 +36,25 @@ It's important to specify which versions of node your application expects. There
 
-### Run `nvm use` before `npm install`
+### Installing packages predictably
 
-You should always ensure you're using the correct version of node (and implicitly `npm`) before doing an `npm install`.
+To reduce "but it works on my machine" issues, use a method that makes installs more predictable, and minimises changes to the `package-lock.json` (if your project is committing that file - this will be the case if your software is a node _application_, but not if it's a _library_).
 
-Running `nvm use` before `npm install` is good because:
-1. It make installs _more_ predictable, which minimises "but it works on my machine" issues.
-1. It minimises changes to the `package-lock.json` (if your project is comitting that file).
+#### Prefer `npm ci` to `npm install` where possible
 
-A problem with using `npm install` is that it doesn't guarantee reproducible builds, as neither the `package.json` nor `package-lock.json` is a source of authority for what's installed.
+The `npm ci` command was introduced with `npm` version 5.7.0+ ([node v10.3+ or 8.12+](https://nodejs.org/en/download/releases/)). 
 
-Better is to use a version of node that ships with `npm` version 5.7.0 or higher ([node v10.3+ or 8.12+](https://nodejs.org/en/download/releases/)). This is because `npm` 5.7.0+ supports the `ci` argument, which is good because **`npm ci` will always install predictably**, using the `package-lock.json` as a source of authority.
+If your project has a `package-lock.json` file (see [Should I check in `package-lock.json` to version control?](#should-i-check-in-package-lockjson-to-version-control)), **`npm ci` will always install predictably**, using the `package-lock.json` as a source of authority. [`npm ci` is also much quicker than `npm install`](https://docs.npmjs.com/cli/ci.html#description) if the `./node_modules` directory is not present (such as in a CI environment).
 
-[`npm ci` is also much quicker than `npm install`](https://docs.npmjs.com/cli/ci.html#description) if the `./node_modules` directory is not present (such as in a CI environment).
+#### nvm users run `nvm use` before `npm install`
 
-(You could use `npm ci` by specifying a newer version of `npm` than is recommended for your [particular version of node](https://nodejs.org/en/download/releases/), but `npm` is itself written in node and [only supports certain node versions](https://github.com/npm/cli/blob/latest/lib/utils/unsupported.js).)
+If you're an nvm user, you should always ensure you're using the correct version of node (and implicitly `npm`) before doing an `npm install`.
 
-#### Automatically running `nvm use`
+If `npm ci` is unavailable (because you're using an older version of npm), or otherwise unsuitable (e.g. you have no `package-lock.json`, or you're installing new/updating existing dependencies), use `npm install`. 
```

Comment:
Same as above, I would change this so it suggests:

* `npm ci` for installing "prod ready" dependencies
* `npm install` to install new dependencies or bump transitive deps

Reply:
Shall we get this PR merged and consider this later as well?